test/ha/activator_test.go

```diff
@@ -38,6 +38,7 @@ import (
 const (
 	activatorDeploymentName = "activator"
 	activatorLabel          = "app=activator"
+	minProbes               = 100  // We want to send at least 100 requests.
```

Comment:
Care to break this change out into a separate PR? If it fixes the SLO flakiness that'd be great and it doesn't seem to have too much in common with the rest of the PR.

Reply:
I'd like to fix even more flakiness in this PR. Is it required to create 2-3 different PRs to fix the same test? I'd rather keep it here.

Reply:
Well, I'd like this to land because the flakiness is really annoying and I mostly see SLO failures.

Reply:
I'm done with this PR (as long as the tests pass). Then we can review/merge. Kenjiro's concern was addressed.